### PR TITLE
Add image bacchelli

### PR DESCRIPTION
**Pull request by justinvdk**
Anything related to boss-level
< feature

### CODE REVIEW

File: src/main/java/nl/tudelft/controller/resources/ResourcesWrapper.java [hunk 1]
```diff
@@ -241,6 +241,10 @@ public Image getBubbleImage6() {
         return images.get("img" + fileSeparator + "rball6.png");
     }
 
+    public Image getBubbleImageBach() {
```
Comment (BCrlrd) [inline src/main/java/nl/tudelft/controller/resources/ResourcesWrapper.java]: Could you add the new image to the ResourcesWrapper test?

< testing

File: src/main/java/nl/tudelft/model/LevelFactory.java [hunk 2]
```diff
@@ -89,11 +90,42 @@ public Level getLevel(int id) throws IllegalArgumentException {
                 return getLevel3();
             case 4:
                 return getLevel4();
+            case 5:
```
Comment (BCrlrd) [inline src/main/java/nl/tudelft/model/LevelFactory.java]: Maybe the boss level should be the 2nd level so Maiko can test it more easily.

< review

File: src/main/java/nl/tudelft/model/LevelFactory.java [hunk 3]
```diff
@@ -89,11 +90,42 @@ public Level getLevel(int id) throws IllegalArgumentException {
...
+        // Create Bubbles for level
+        LinkedList<AbstractBubble> bubbles = new LinkedList<>();
+
+        AbstractBubble bubble = new BacchelliBubble(resources, 200,
+                resources.getVwallImage().getWidth() );
+
+        bubbles.add(bubble);
+
+        LinkedList<AbstractWall> walls = wallInit();
+        {
```
Comment (BCrlrd) [inline src/main/java/nl/tudelft/model/LevelFactory.java]: What's the point of these brackets?
Can this commented code be deleted?

< review

File: src/main/java/nl/tudelft/model/LevelFactory.java (hunk 2 above)
Comment (justinvdk) [inline src/main/java/nl/tudelft/model/LevelFactory.java]: Will put boss level as second level for release 1.5

File: src/main/java/nl/tudelft/model/LevelFactory.java (hunk 3 above)
Comment (justinvdk) [inline src/main/java/nl/tudelft/model/LevelFactory.java]: The brackets were there for structure. This code is removed as it is no longer needed.

File: src/main/java/nl/tudelft/controller/resources/ResourcesWrapper.java (hunk 1 above)
Comment (justinvdk) [inline src/main/java/nl/tudelft/controller/resources/ResourcesWrapper.java]: added the bacc image to the resourcewrapper test < testing

**Pull request merged**